src/ray/protobuf/gcs_service.proto

```diff
@@ -39,6 +39,9 @@ message MarkJobFinishedReply {
 message GetAllJobInfoRequest {
   // The number of jobs to return. If not specified, return all jobs.
   optional int32 limit = 1;
+  // If set, only return the job with that job id in hex, or the job with that
+  // job_submission_id.
+  optional bytes job_or_submission_id = 3;
```

Comment:
Let's make this repeated to make this proper batch API

Reply:
I would refrain until we find a need for it